File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--add-content-controls-to-word-documents_1.cs
        private Microsoft.Office.Tools.Word.RichTextContentControl richTextControl1;

        private void AddRichTextControlAtSelection()
        {
            this.Paragraphs[1].Range.InsertParagraphBefore();
            this.Paragraphs[1].Range.Select();

            richTextControl1 = this.Controls.AddRichTextContentControl("richTextControl1");
            richTextControl1.PlaceholderText = "Enter your first name";
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--add-content-controls-to-word-documents_1.vb
    Dim richTextControl1 As Microsoft.Office.Tools.Word.RichTextContentControl

    Private Sub AddRichTextControlAtSelection()
        Me.Paragraphs(1).Range.InsertParagraphBefore()
        Me.Paragraphs(1).Range.Select()
        richTextControl1 = Me.Controls.AddRichTextContentControl("richTextControl1")
        richTextControl1.PlaceholderText = "Enter your first name"
    End Sub